src/lib/core/platform/browser.ts

```diff
@@ -0,0 +1,4 @@
+/** Gets whether the current platform is a web browser (versus node). */
+export function isBrowser() {
+  return typeof document === 'object';
```

Comment:
Changed check to `typeof document === 'object' && document;`

We can get other places in a follow-up.